src/pcsswebservice/pom.xml

```diff
@@ -9,10 +9,10 @@
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>ca.bc.gov.open.pcss</groupId>
-	<artifactId>pcss-criminal-api</artifactId>
+	<artifactId>pcsswebservice</artifactId>
```

Comment:
can we name it pcss-web-service ? you just have to change the artifact id and the folder name

Reply:
Done!